src/main/java/org/myrobotlab/service/data/Locale.java

```diff
@@ -143,15 +143,30 @@ public String getDisplayCountry() {
   }
 
   public static Map<String, Locale> getDefaults() {
-
+// Pulls all languages available from the OS, not useful to us
+//    Map<String, Locale> locales = new TreeMap<>();
+//    java.util.Locale[] ls = java.util.Locale.getAvailableLocales();
+//    for (java.util.Locale l : ls) {
+//      Locale newLocale = new Locale(l.toString());
+//      if (l.toString() != null && l.toString().length() != 0) {
+//        locales.put(newLocale.tag, newLocale);
+//      }
+//    }
+    // We really only support a few Locales dictated by ProgramAB, Polly,
+    // WebkitSpeechRecognition, & WebKitSpeechSynthesis
     Map<String, Locale> locales = new TreeMap<>();
-    java.util.Locale[] ls = java.util.Locale.getAvailableLocales();
-    for (java.util.Locale l : ls) {
-      Locale newLocale = new Locale(l.toString());
-      if (l.toString() != null && l.toString().length() != 0) {
-        locales.put(newLocale.tag, newLocale);
-      }
-    }
+    locales.put("cn-ZH", new Locale("cn-ZH"));
+    locales.put("de-DE", new Locale("de-DE"));
+    locales.put("en-US", new Locale("en-US"));
+    // locales.put("en-GB", new Locale("en-GB"));
+    locales.put("fi-FI", new Locale("fi-FI"));
+    locales.put("fr-FR", new Locale("en-FR"));
+    locales.put("hi-IN", new Locale("hi-IN"));
+    locales.put("it-IT", new Locale("it-IT"));
+    locales.put("nl-NL", new Locale("nl-NL"));
+    locales.put("pt-PT", new Locale("pt-PT"));
+    locales.put("ru-RU", new Locale("ru-RU"));
+    locales.put("tr-TR", new Locale("tr-TR"));
```

Comment:
Not sure this is what we should do, what about dialects like `en-GB`? Was there a particular issue with the auto detection that hard coding fixes?

Reply:
auto detection is "auto" detecting the OS
which is not relevant to the locales the services provide.
It made choosing a language difficult in the UI, where this is the only use it provides.
Perhaps a better implementation is to merge all running services sets into one set..    which could be a different pr